File: src/main/java/com/ciazhar/authserver/repository/RoleRepository.java
package com.ciazhar.authserver.repository;

import com.ciazhar.authserver.model.jpa.Role;
import org.springframework.data.repository.PagingAndSortingRepository;

/**
 * Created by ciazhar on 6/23/17.
 */
public interface RoleRepository extends PagingAndSortingRepository<Role,String> {
}

File: src/main/kotlin/com/ciazhar/authserver/repository/RoleRepository.kt
package com.ciazhar.authserver.repository

import com.ciazhar.authserver.model.jpa.Role
import org.springframework.data.repository.PagingAndSortingRepository

/**
 * Created by ciazhar on 6/23/17.
 */
interface RoleRepository : PagingAndSortingRepository<Role, String>
